source/security/mosca-tls.rst

```diff
@@ -116,91 +102,58 @@ In order to create the CSR file and ask for a certificate signature, a user can
 use a helper script called 'Certificate Retriever', which is detailed in
 `Certificate retriever`_ section.
 
-MQTT Manager
+Mosca
 ~~~~~~~~~~~~
-
-MQTT-Manager is a helper service used to configure Mosquitto MQTT broker in a
-simple and 'on-the-fly' way. It can be configured using REST interfaces and
-Kakfa. Thus, HTTP requests or Kafka messages can be used to create and remove
-devices, as well as update CRL file (certification revocation list). This
-service is distributed as a docker container for easy deploy and its source
-code repository can be accessed in `MQTT Manager repository`_.
-
-Mosquitto by itself doesn't generate nor revoke certificates, it only relies on
-a CA and implements TLS protocol. The 'creation' of a particular device
-consists only in adding a new rule to ACL file in Mosquitto. Such file looks
-like:
-
-.. code:: ini
-
-    user iotagent
-    topic read /#
-    user 24f6
-    topic write /admin/24f6/attrs
-
-Each rule is composed by two lines: the first one specifies the user (device)
-and the second one defines which action (write or read) is allowed to which
-topic. In the example above, the user iotagent can read all topics (# is a
-wildcard). Also, the device with ID 24f6 can write to topic /admin/24f6/attrs.
-The device ID is retrieved in 'Common name' certificate field.
-
-If a device sends data to a topic which it has no write permissions, then all
-data is discarded. Mosquitto won't log any errors related to this.
-
-When the ACL is changes, Mosquitto must be restarted (or a SIGDUP signal can be
-sent to its process). MQTT-Manager does this automatically when creating or
-removing devices.
-
-A script is executed when firing the container up. This script will generate a
-pair of keys to Mosquitto, retrieves the certificate and CRL from a CA and asks
-it to sign its public key. ALl generated files are placed in
-/usr/local/src/mosquitto-1.4.13/certs (inside the container).
-
-Mosquitto will only accept device connections that have certificate signed by
-its trusty CA.
-
-Also note that MQTT-Manager is used only in case when a TLS-enabled broker is
-needed. If this is not the case, then the vanilla `Mosquitto docker image`_ can
-be used.
-
-Mosquitto configuration files
+Mosca is a node.js mqtt broker. To use the Mosca broker with TLS, you need to allow the broker
+to use TLS as the secure mode of connection. To do this, you need to declare the credentials and
+the secure attributes in the Mosca server conf object.
+
+.. code-block:: JavaScript
+
+    ...
+    type : "mqtts", // important to only use mqtts, not mqtt
```

Comment:
As I said in dojot/iotagent-mosca#14, this might be better placed in a config file or environment variables (or both).